lib/server.js

```diff
@@ -191,7 +191,7 @@ Server.prototype.close = function () {
   if (this.ws) {
     debug('closing webSocketServer');
     this.ws.close();
-    delete this.ws;
+    // don't delete this.ws because it can be used again if the server starts listening again
```

Comment:
I understand the purpose here, but won't the comment seem a bit weird since the line `delete this.ws;` will be deleted?